bcipy/task/paradigm/rsvp/copy_phrase.py

```diff
@@ -503,11 +505,15 @@ def compute_eeg_evidence(self,
         if not proceed or self.fake:
             return None
 
-        raw_data, triggers, labels = get_data_for_decision(
+        raw_data, triggers = get_data_for_decision(
             inquiry_timing=self.stims_for_decision(stim_times),
             daq=self.daq,
-            static_offset=self.parameters['static_trigger_offset'],
-            buffer_length=self.parameters['trial_length'])
+            offset=self.parameters['static_trigger_offset'],
+            prestim=self.parameters['prestim_length'],
+            poststim=self.parameters['task_buffer_length'] + self.parameters['trial_length'])
+
+        # we assume all are nontargets at this point
+        labels = ['nontarget'] * len(triggers)
```

Comment:
Why assume that these are all non-targets? Is this a TODO item? The `task` helper module has a `target_info` function for computing this given the triggers and target symbol.

Reply:
We don't need these labels. Not sure why they were added before... I created a ticket to address this since it could get messy! https://www.pivotaltracker.com/story/show/181861101

Reply:
Just chatted with Tab. Since we're just getting model predictions, presumably it should not matter what the labels are. Tab made a ticket to remove labels completely from this function: https://www.pivotaltracker.com/story/show/181861101